calico_node/filesystem/etc/calico/confd/templates/bird6.cfg.template

```diff
@@ -1,6 +1,5 @@
 # Generated by confd
 include "bird6_aggr.cfg";
-include "custom_filters6.cfg";
```

Comment:
Have we just removed support for these?

Reply:
(Sounds like we have - @robbrockbank to make a backlog task)

Reply:
i've added a backlog task, yup